package.json

```diff
@@ -19,11 +19,11 @@
     "start-with-local-graphql-and-article-converter": "cross-env LOCAL_GRAPHQL_API=true RAZZLE_LOCAL_ARTICLE_CONVERTER=true razzle start",
     "start-with-local-article-converter": "cross-env RAZZLE_LOCAL_ARTICLE_CONVERTER=true yarn start",
     "start-prod": "cross-env NODE_ENV=production node build/server | bunyan",
-    "e2e": "cypress open",
-    "e2e-use-fixtures": "cross-env CYPRESS_USE_FIXTURES=true cypress open",
-    "e2e-record-fixtures": "cross-env CYPRESS_RECORD_FIXTURES=true cypress run",
+    "e2e": "BABEL_ENV=test cypress open",
```

Comment:
Hva skal denne være? BABEL_ENV eller NODE_ENV må være satt i cypress 6

Reply:
Tja. Kva seier cypressdokumentasjonen?

Reply:
Ser ut som det er en bug faktisk: https://github.com/cypress-io/cypress/issues/6755. Kan kanskje la det stå som det er inntil videre?

Reply:
Tror kanskje vi må bruke cross-env for å sette denne for at det skal fungere likt på windows?